src/bci_build/package/__init__.py

```diff
@@ -350,6 +344,8 @@ def __post_init__(self) -> None:
                 self.exclusive_arch = [Arch.AARCH64, Arch.X86_64]
             # Disable maintainer listing
             self.maintainer = None
+        elif not self.maintainer and self.os_version.is_sle15:
+            self.maintainer = "https://github.com/SUSE/bci/discussions"
```

Comment:
This means that the maintainer is not set for SLFO, I guess that's not intentional?

Reply:
it was intentional. I don't see a lot of value in the field anymore and it is marked as optional in the spec. I think we should rather set it on the images that actually have a dedicated maintainer/forum instead of everywhere, including the base containers. 

but I'm not super opinionated on this. all we know is that we have the directive to not drop labels on sle15, so first where we can do the change is tumbleweed/slfo

Reply:
I'd be fine with dropping it only for openSUSE if that's what is being preferred.

Reply:
I've changed it to "not for openSUSE" to address the concern and rebased to avoid the merge conflict.